### PR TITLE
moving pytest to requirements-dev

### PR DESCRIPTION
**Pull request by JeremyDrysdale-ibm**
### What are the issues this pull addresses (issue numbers / links)?
None

### Did you add tests to cover your changes (yes/no)?
No

### Did you update the documentation accordingly (yes/no)?
No

### Did you read the CONTRIBUTING document (yes/no)?
No

### Summary
moving pytest to requirements-dev

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,6 +71,11 @@ conda env create -n <env_name> environment.yml
 conda activate <env_name>
 python -m pip install -ve .
 ```
+Execut the following command if you plan to contribute code:
```
Comment (priti-ashvin-shah-ibm) [inline README.md:74]: Execute

File: README.md [hunk 2]
```diff
@@ -86,6 +91,11 @@ conda env update -n <env_name_exist> environment.yml
 conda activate <env_name_exist>
 python -m pip install -ve .
 ```
+Execut the following command if you plan to contribute code:
```
Comment (priti-ashvin-shah-ibm) [inline README.md:94]: Execute

Comment (marcolincs) [reply README.md]: both lines 74 and 94

**Review by marcolincs — APPROVED** [review]
LGTM

**Review by priti-ashvin-shah-ibm — APPROVED** [review]
LGTM

**Pull request merged**